cmd/nerdctl/container_run_soci_linux_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestRunSoci(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	base := testutil.NewBase(t)
+	requiresSoci(base)
+
+	initialMounts, err := exec.Command("mount").Output()
+	if err != nil {
+		t.Fatalf(err.Error())
+	}
+
+	remoteSnapshotsInitialCount := strings.Count(string(initialMounts), "fuse.rawBridge")
+	base.T.Logf("number of initial mounts: %v", remoteSnapshotsInitialCount)
+
+	if remoteSnapshotsInitialCount != 0 {
+		t.Fatalf("initial mounts count isn't zero")
+	}
+
+	//validating that run works when unpackB is true and soci is specified
```

Comment:
Where is unpackB? Didn't see in your command.

Reply:
Specifying a platform when running should make unpackB true https://github.com/containerd/nerdctl/blob/8cef5e9166c76303f444bd36001e4b6eadaa19e3/pkg/imgutil/imgutil.go#L214C2-L215C39

Reply:
Maybe also link here?

Reply:
<img width="663" alt="Screenshot 2023-07-21 at 5 35 42 PM" src="https://github.com/containerd/nerdctl/assets/8747309/faf286f7-1c11-4998-bebc-e7fddc3d1538">
Make sure you copy the permalink for every place.